docs/variables.md

```diff
@@ -754,6 +791,20 @@ not.
 
 Specifies the name of the column family to compact.
 
+### `rocksdb_compact_lzero_now`
+
+| Option       | Description          |
+|--------------|----------------------|
+| Command-line | --rocksdb-compact-lzero-now |
+| Dynamic      | Yes                  |
+| Scope        | Global               |
+| Data type    | Boolean              |
+| Default      | OFF                  |
+
+This variable has been implemented in [Percona Server for MySQL 8.0.36-28](.//release-notes/8.0.36-28.md).
+
+This variable triggers a compaction of all `L0` files in RocksDB. The default value is `OFF`, which means the variable is disabled.
```

Comment:
This variable is a switch. We call `rocksdb-compact-lzero-now=ON` to immediately compact L0 files and later it is automatically set to `OFF`.

Reply:
updated